redaxo/src/core/lib/console/package/activate.php

```diff
@@ -22,6 +22,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
     {
         $io = $this->getStyle($input, $output);
 
+        /** @var string $packageId */
```

Comment:
Kann man vllt an der methode was deklarieren, dass bei key=package-id nen string returned wird?

Reply:
Die Methode ist ja in symfony definiert

Reply:
Dann vllt über das pphstorm meta file? Das wird von psalm ja auch gelesen

Reply:
aber allgemein für package-id wäre es falsch, es hängt ja vom jeweiligen command hab, wie das argument oben definitiert ist.
kann ja auch sein, dass wir in einem command mal package-id als optionales argument haben, dann kann auch null zurück kommen.

Reply:
man müsste eher ein psalm plugin schreiben, was die definition von oben analysiert. das ist mir aber zu kompliziert.

Reply:
Ich habe dazu ein Issue angelegt: https://github.com/psalm/psalm-plugin-symfony/issues/17